system/lib/libc/musl/src/math/rint.c

```diff
@@ -11,6 +11,10 @@ static const double_t toint = 1/EPS;
 
 double rint(double x)
 {
+// XXX EMSCRIPTEN: on wasm backend, use the wasm instruction via clang builtin
```

Comment:
Perhaps you could add a link to the bug for more context

Reply:
Done.